src/mame/apple/tk10_keyboard.h

```diff
@@ -0,0 +1,51 @@
+// license:BSD-3-Clause
+// copyright-holders: Golden Child
+
+#ifndef MAME_TK10_KEYBOARD_H
+#define MAME_TK10_KEYBOARD_H
```

Comment:
This will fail CI now that `#include` guards are checked.

Reply:
ok, so I just have to rename it so it matches the filename

MAME_TK10_KEYBOARD_H
to
MAME_APPLE_TK10_KEYBOARD_H

so that the include guard has the proper name like:

MAME_APPLE_APPLE2COMMON_H